go/sqltypes/result.go

```diff
@@ -117,3 +120,45 @@ func (result *Result) StripFieldNames() *Result {
 	}
 	return &r
 }
+
+// AppendResult will combine the Results Objects of one result
+// to another result.Note currently it doesn't handle cases like
+// if two results have different fields.We will enhance this function.
+func (result *Result) AppendResult(qr, innerqr *Result) {
```

Comment:
This should be `func (result *Result) Append(src *Result)`, where `src` is appended to `result`. The function name doesn't need to contain Result because it's implicit.